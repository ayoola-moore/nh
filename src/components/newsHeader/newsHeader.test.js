import React from "react";
import NewsHeader from "./";

describe("Component", () => {
  const component = shallow(<NewsHeader />);
  it("should render FeedContainerWrapper properly", () => {
    expect(component).toMatchSnapshot();
  });
});
