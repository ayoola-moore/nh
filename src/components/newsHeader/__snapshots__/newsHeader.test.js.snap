// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should render FeedContainerWrapper properly 1`] = `ShallowWrapper {}`;
